frontend/src/components/NewRunParametersV2.test.tsx

```diff
@@ -484,7 +484,7 @@ describe('NewRunParametersV2', () => {
     screen.getByDisplayValue('"A":1,"B":2');
   });
 
-  it('test Start button is enabled for valid default integer input', () => {
+  it('return true to parent component with valid default integer input', () => {
```

Comment:
The term `return true` is a bit ambiguous to learn what this test means. Can we use the following term? `set input as valid type with default integer parameter`.